packages/code-du-travail-data/dataset/preavis-licenciement/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
```

Comment:
changelog are generated... you should not add it

Reply:
OK